packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/views/ViewManagerWrapperDelegate.kt

```diff
@@ -32,6 +32,7 @@ class ViewManagerWrapperDelegate(internal var moduleHolder: ModuleHolder) {
 
   fun setProxiedProperties(view: View, proxiedProperties: ReadableMap) {
     definition.setProps(proxiedProperties, view)
+    definition.onViewDidUpdateProps?.invoke(view)
```

Comment:
Does it work for Fabric too?

Reply:
I didn't test that, but it seems it will work for Fabric. On the native side, we have only one implementation of the view manager that is used on both Paper and Fabric. That implementation proxy everything via `ProxiedProperties`. So that function should be called in both cases.